lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -3,24 +3,25 @@ module Compute
     class Ovirt
       class V4
         class Real
-          def check_for_option(opts, name)
-            opts[name.to_sym] || opts[(name + "_name").to_sym]
-          end
 
           # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
With all those changes, the method looks quite slim. Can we readd those rubocop rules back?